src/api/archive_unstable_storageDiffContinue.md

```diff
@@ -0,0 +1,13 @@
+# archive_unstable_storageDiffContinue
```

Comment:
I wonder about calling it `archive_unstable_continue` to align with the chainHead one?

Reply:
Made the subscription a method after all, thanks